parkhangs-expressapp/database/index.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose');
+
+let DBNAME = "ParkHangs";
+let PASSWORD = "PUT DB PASSWORD HERE";
```

Comment:
we should have the password in a .env file that we don't commit to github but have locally so we don't have to change this file every time we pull from master. so instead it's like `const PASSWORD = process.env.MONGO_PASSWORD`

Reply:
Yes, in the new commit, I change it so it will use an .env file. Each of you need to make a .env file in the parkhangs-expressapp folder and npm install dotenv